openassessment/xblock/openassessmentblock.py

```diff
@@ -1186,6 +1186,8 @@ def get_real_user(self, anonymous_user_id):
 
         """
         if hasattr(self, "xmodule_runtime"):
+            if self.xmodule_runtime.get_real_user is None:  # pylint: disable=no-member
```

Comment:
I was running into this on studio, when the page is rendered as you are adding assignments. I assumed that was what `is_studio_preview` is, but that check was returning false? Anyways, the above line was true but `self.xmodule_runtime.get_real_user` was `None` so the call was blowing up. 🤷‍♂